src/Geopilot.Api/Program.cs

```diff
@@ -56,7 +56,7 @@
     .AddJwtBearer(JwtBearerDefaults.AuthenticationScheme, options =>
     {
         options.Authority = builder.Configuration["Auth:Authority"];
-        options.Audience = builder.Configuration["Auth:ClientId"];
+        options.Audience = builder.Configuration["Auth:ApiId"];
```

Comment:
REQ: In der Konfig wird nicht klar, wofür das verwendet wird. Bitte etwas wie `Auth:Audience` oder `Auth:TokenAudience` verwenden.